### PR TITLE
25 14 admin invoices index page

### LINKED ISSUE #25: #14 - Admin Invoices Index Page
As an admin,
When I visit the admin Invoices index ("/admin/invoices")
Then I see a list of all Invoice ids in the system
Each id links to the admin invoice show page

### PR DESCRIPTION
**Pull request by LlamaBack**
just listing out admin invoices in the index

### CODE REVIEW

File: app/views/admin/invoices/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<h1>All Invoices</h1>
+
+<% @invoices.each do |invoice| %>
+  <h3>Invoice #<%= link_to invoice.id, admin_invoice_path(invoice) %></h3>
+<% end %>
```
Comment (Astrid-Hecht) [inline app/views/admin/invoices/index.html.erb:5]: We should make sure we figure out a way to handle the header/nav bar in the app layout. I've been writing them into to the views but I can go back and fix them once we have that figured out.

**Review by Astrid-Hecht — COMMENTED** [review]
Looks good!

File: app/views/admin/invoices/index.html.erb (hunk 1 above)
Comment (LlamaBack) [reply app/views/admin/invoices/index.html.erb:5]: sounds good. please teach me what to do!!

**Review by ajkrumholz — APPROVED** [review]
Looks good!

**Review by Alaina-Noel — APPROVED** [review]

**Pull request merged**